.github/workflows/generate-timestamp.yaml

```diff
@@ -0,0 +1,42 @@
+name: Generate a timestamp
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+
+jobs:
+  generate-timestamp:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout scalar-licensing repo
+        uses: actions/checkout@v3
+        with:
+          repository: scalar-labs/scalar-licensing
+          token: ${{ secrets.TIMESTAMP_GH_PAT }}
+
+      - name: Setup and execute Gradle 'installDist' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: installDist
+
+      - name: Generate a timestamp
+        run: |
+          echo "${{ secrets.PRIVATE_KEY_PEM }}" > $HOME/private-key.pem
+          lib/build/install/lib/bin/generate-signed-timestamp --private-key-file=$HOME/private-key.pem > $HOME/timestamp.json
+
+      - name: Checkout this repo
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.TIMESTAMP_GH_PAT }}
+
+      - name: Commit the timestamp
+        run: |
+          cp $HOME/timestamp.json .
+          git config user.name github-actions
+          git config user.email action@github.com
+          git add .
+          git commit -m "Update the timestamp"
```

Comment:
If we add a commit each time we update the timestamp, the history of timestamps accumulates, and a malicious user could potentially reuse them. How about overwriting the previous commit by executing a `git commit` with the `--amend` option to eliminate this history?

Reply:
That's a good idea! Thanks!

Reply:
@brfrn169 Updated in [60211b2](https://github.com/scalar-labs/timestamp/pull/1/commits/60211b28d9e58f6b1cea472655d8c27bc70767f3).